core-bundle/src/EventListener/TwoFactorFrontendListener.php

```diff
@@ -24,9 +24,6 @@
 use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
 use Symfony\Component\Security\Http\Util\TargetPathTrait;
 
-/**
- * @internal
```

Comment:
All event listener are internal, so this change probably needs to be reverted.

Reply:
Fixed in 1d6ab3b7b74e40facc7950ce3b7fdabfdf68c80d